agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/statsbeat/FeatureStatsbeat.java

```diff
@@ -129,9 +129,15 @@ void trackConfigurationOptions(Configuration config) {
     }
 
     // preview instrumentation
+    if (!config.preview.instrumentation.apacheCamel.enabled) {
```

Comment:
again, just following convention, let me know if you want to change it for existing too

Reply:
feature tracking is based on default value.. if it's off by default, we track on; on by default, we track off.. 
in this case, DisabledByDefaultInstrumentation  is off by default.. here only check for false, which seems wrong.

Reply:
I agree what you suggest would be better, but that's not how current features work, do you want me to change the existing convention?

Reply:
i think it's because moving config from instrumentation to preview make it confusing.. under instrumentation, it's on by default.. now preview default is off..  once it moves to preview, we change the default value.. that's why feature needs to be updated as well to reflect this change.

Reply:
@heyams please send a commit to this PR to update statsbeat the way you want it